packages/react/src/hooks/useTemplateFeatureEnabledFor copy.ts

```diff
@@ -0,0 +1,77 @@
+import { useEffect, useState } from "react";
+import { Fetcher } from "@carrot-kpi/sdk";
+import { useNetwork, type Address, usePublicClient } from "wagmi";
+import { usePreferDecentralization } from "./usePreferDecentralization";
+
+interface TemplateFeatureEnabledForParams {
+    type?: "oracle" | "kpiToken";
+    templateId?: number;
+    featureId?: number;
+    account?: Address;
+}
+
+export function useTemplateFeatureEnabledFor(
+    params?: TemplateFeatureEnabledForParams,
+): {
+    loading: boolean;
+    allowed: boolean;
```

Comment:
I'd rename `allowed` to `enabled` to be consistent with the function name, or the opposite, wdyt?

Reply:
Should be addressed in 3bb57910d803184cb4385d2b2b7f1707c3413123